tests/micro/arduino/test_arduino_rpc_server.py

```diff
@@ -56,30 +59,50 @@ def _make_session(model, arduino_board, arduino_cli_cmd, workspace_dir, mod, bui
 
 
 def _make_sess_from_op(
-    model, arduino_board, arduino_cli_cmd, workspace_dir, op_name, sched, arg_bufs, build_config
+    model,
+    arduino_board,
+    arduino_cli_cmd,
+    workspace_dir,
+    op_name,
+    sched,
+    arg_bufs,
+    build_config,
+    serial_number: str,
```

Comment:
Fix type annotation to reflect that `serial_number` is optional. This happens in a few other tests as well.

Reply:
done